modules/ibi-ib-cli-reference.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_content-type: REFERENCE
+[id="ibi-ib-cli-reference_{context}"]
+= ib-cli CLI tool API specifications
+
+The `ib-cli` CLI tool uses the `image-based-install-iso.yaml` file to create a live installation ISO. The following content describes the specifications for the `image-based-install-iso.yaml`. 
+
+.Required specifications
+[options="header"]
+[cols="2a,1a,4a"]
+|====
+|Specification|Type|Description
+|`seedImage`|`string`|Specifies the seed image to use in the ISO generation process.
+|`seedVersion`|`string`|Specifies the {product-title} release version of the seed image.
+|`installationDisk`|`string`|Specifies the disk that will be used for the installation process. 
+
+Because the disk discovery order is not guaranteed, the kernel name of the disk can change across booting options for machines with multiple disks. For example, `/dev/sda` becomes `/dev/sdb` and vice versa. To avoid this issue, you must use a persistent disk attribute, such as the disk World Wide Name (WWN), for example: `/dev/disk/by-id/wwn-<disk-id>`.
+
+|====
+
+.Optional specifications
+[options="header"]
+[cols="2a,1a,4a"]
+|====
+|Specification|Type|Description
+|`pullSecret`|`string`|Specifies the pull secret to use during the precache process. The pull secret contains authentication credentials for pulling the release payload images from the container registry.
+|`shutdown`|`boolean`|Specifies if the host shuts down after the installation process completes.
+|`extraPartitionStart`|`string`|Specifies the start of the extra partition used for `/var/lib/containers`. The partition will expand until the end of the disk. Uses sgdisk notation.
+|`ExtraPartitionLabel`|`string`|???
```

Comment:
maybe better to remove it? @javipolo i don't see why user should change default and you?
Same with ExtraPartitionNumber